Sming/sming.mk

```diff
@@ -0,0 +1,662 @@
+#   sming.mk
+#
+#   to do:
```

Comment:
At some point take care to remove all comments from the top of the makefile and add them to the commit history.